### DISCUSSION TITLE
Home Assistant OS 11.0.rc1

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#2793] [Announcements]
Home Assistant OS 11.0.rc1 is built from Buildroot version 2023.02.05. With this release Home Assistant OS uses Docker 24.0.6 and containerd 1.7.6.

Most notable changes are:
  * Using Linux' Preemptible Kernel configuration across all boards (lower latencies even on busy systems)
  * Filesystem freeze is relayed to database engines on Proxmox/KVM based VMs (this makes VM snapshots always coherent)
  * Various Bluetooth improvements (newer BlueZ, better fix Bluetooth LE advertisement stalls, Bluetooth device cache management)
  * Improved Kernel configuration should improve Docker's overlayfs performance
  * Improved containerd build and configuration makes containerd use less memory and CPU resources
  * Adjusted development work-flow which allows more incremental changes

## Changes

* Improve grammar in README.md (#2790) @davet2001
* RAUC update: Use verity bundle format (#2786) @agners
* Bump docker/login-action from 2.2.0 to 3.0.0 (#2757) @dependabot
* Bump docker/build-push-action from 4.2.1 to 5.0.0 (#2758) @dependabot
* Bump docker/setup-buildx-action from 2.10.0 to 3.0.0 (#2759) @dependabot
* Bump docker/build-push-action from 4.1.1 to 4.2.1 (#2748) @dependabot
* Bump buildroot to 2023.02.4 (#2737) @agners
* Bump actions/checkout from 3 to 4 (#2728) @dependabot
* Bump docker/login-action from 2.1.0 to 2.2.0 (#2727) @dependabot
* Remove checkout twice in dev build workflow (#2719) @adeepn

## Home Assistant Operating System

* Set containerd root explicitly (#2476) (#2791) @agners
* Remove unnecessary plug-ins from containerd (#2789) @agners
* Bump buildroot to 2023.02.5 (#2775) @agners
* Bump buildroot to update Docker to v24.0.6 (#2774) @agners
* Disable overlayfs features not used by Docker (#2773) @agners
* Update and fix containerd v1.6.22 (#2770) @agners
* Adjust Home Assistant OS versioning to prepare for new release strategy (#2767) @agners
* Download latest stable Supervisor after device wipe (#2765) @agners
* Use latest Network device naming scheme (#2763) @agners
* Create swapfile even when not using the multi-user.target (#2762) @agners
* Delete Bluetooth device cache regularly (#2751) @agners
* Bump buildroot to update go to v1.19.13 (#2749) @agners
* Bump buildroot to revert containerd to v1.6.21 (#2744) @agners
* Bump HAOS Agent to 1.6.0 (#2734) @agners
* Improve LED representation in Home Assistant Green device tree (#2730) @agners
* Use full preemptible kernel in Home Assistant OS (#2721) @agners
* Bump buildroot to update BlueZ (#2715) @agners
* Use proper fix for Bluetooth LE advertisement stall (#2598) (#2714) @agners
* Add tools for SPI NOR flash storage (#2696) @agners
* Update U-Boot patches and configuration (#2694) @agners

## Build System

* Add action job to upload artifacts to R2 (#2779) @ludeeus
* Fix versioning set during build (#2771) @agners
* Adjust Home Assistant OS versioning to prepare for new release strategy (#2767) @agners
* Unify release and dev GitHub actions (#2764) @agners
* Don't combine GitHub action caches (#2745) @agners
* Combine downloads and compile cache in GitHub Action cache (#2743) @agners
* Execute stale workflow only once a day (#2739) @agners
* Improve GitHub Action caching (#2732) @agners
* Separate build step when loading container images (#2731) @agners
* configs: Enable LTO if available (#2713) @agners
* GitHub action improvements and cleanup (#2710) @agners
* Create builder group only if required (#2709) @agners

## Open Virtual Appliance

* Fix fsfreeze support on OVA (#2787) @agners
* Add fsfreeze support for QEMU/KVM/Proxmox installations (#2781) @agners
* Linux: Update kernel 6.1.55 (#2769) @agners
* Bump buildroot to update openvmtools (#2733) (#2735) @agners

## Generic x86-64

* generic-x86-64: enable backlight support (#2772) @craftyguy
* Linux: Update kernel 6.1.55 (#2769) @agners
* Enable ECC memory support via EDAC (#2708) (#2736) @agners

## Hardkernel ODROID

* Linux: Update kernel 6.1.55 (#2769) @agners

## ASUS Tinker

* Linux: Update kernel 6.1.55 (#2769) @agners

## Khadas VIM Series

* Linux: Update kernel 6.1.55 (#2769) @agners

## Generic aarch64

* Linux: Update kernel 6.1.55 (#2769) @agners

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/11.0.rc1'>Home Assistant OS 11.0.rc1</a>.</em>

**Comment by tjorim** [comment]
For those wondering, the upgrade went fine on my HA Yellow with a nvme data disk.

**Comment by bschatzow** [comment]
Upgrade for me on RPI 4 with SSD had no issues.

**Comment by gieljnssns** [comment]
Upgrade on a nuc also had no issues.

**Comment by rennokki** [comment]
To some degree, https://github.com/home-assistant/operating-system/issues/2592#issuecomment-1589369915 is still present even with the pybluez update.

**Comment by ssbarnea** [comment]
It would be quite useful to document which image we could use to import into Parallel Desktop running on M1/M2 macs. For some reason conversion of the vmware image does not seem to work and the OVA images seem to not have the architecture mentioned, which makes me believe is might not be aarch64.

**Comment by pimw1** [comment]
I've very positive feedback to mention: cpu utilization went down from 12%-16% to 6%-10%. This is on a RPI4 with SSD, comparing 10.5 to 11.0.rc1.
Since aprox a year ago with one of the previous OS version upgrades, cpu utilization went up and the root cause never became clear. I'm happy to see that this is resolved now.

**Comment by yousaf465** [comment]
![image](https://github.com/home-assistant/operating-system/assets/83491212/ca18f692-cb10-41dc-8351-838144007626)
Since installing HAOS 11.0.rc1, my Homeassistnt is rebooting every 15 minutes. 

my current system is rpi4 2Gb with the following home assistant specs. 
```
Home Assistant 2023.10.0
Supervisor 2023.10.0
Operating System 11.0.rc1
Frontend 20231002.0 - latest
```

also noticed an steady increase in storage usage 
![image](https://github.com/home-assistant/operating-system/assets/83491212/fc3022ca-9092-4436-9d9d-24be1fad7d6f)

**Reply by N3rdix** [reply]
Have you checked if tons of log entries are causing this issue? I could not see an increase in storage usage on my Pi test landscape

**Comment by N3rdix** [comment]
Although Raspberry Pi's seem to not have been affected by this issue I checked 11.0rc1 on a test environment and it still works flawlessly. No CPU increase, pretty much the same as with 9.5 / 10.5
Awesome work @agners 👍
